src/components/Banner/MucBanner.vue

```diff
@@ -6,12 +6,22 @@ import { computed } from "vue";
 type bannerType = "info" | "warning" | "emergency";
```

Comment:
I overlooked the patternlab-page and found that a [fourth type](https://patternlab.muenchen.space/?p=elements-bb-banner) of banner exists. Maybe we could add this type too in this overhaul.

Reply:
I thought about that, but the specific component is only used once on the whole page and wouldn't have much of an impact in custom webcomponents.

Reply:
Fair enough - thought about the same. Just wanted to bring this up.